### PR TITLE
chore: Removed unused scripts based on new release process

### PR DESCRIPTION
**Pull request by abhiomkar**

### CODE REVIEW

File: scripts/post-release.sh [hunk 1]
```diff
@@ -28,27 +28,12 @@ function log() {
   echo '\033[36m[post-release]\033[0m' "$@"
 }
 
-if [[ $(git diff --cached CHANGELOG.md) ]]; then
```
Comment (moog16) [inline scripts/post-release.sh]: when/how are we supposed to generate the changelog instead?

Comment (abhiomkar) [reply scripts/post-release.sh]: Deleted this script & cherry pick script that are unused based on new release process documented internally.

We would need to run `npm run changelog` log command to generate changelog content as per new release process documentation.

Comment (moog16) [reply scripts/post-release.sh]: Do you also want to delete this repo's release process doc in this PR then?

Also I don't see running the post-release script in the new doc.

Comment (abhiomkar) [reply scripts/post-release.sh]: Updated this PR to delete cherry pick script and release process doc.

**Pull request merged**